docs/source/installation.rst

```diff
@@ -568,29 +568,54 @@ Manual Installation
     $ source $PATH_TO_VENV/bin/activate
     (.mozdef_env)$ cd $MOZDEF_PATH/examples/es-docs && python inject.py
 
+10. Installing Supervisord to enable Alerting on events.
+     
+    $ sudo -i -u mozdef -g mozdef
+    $ cd /home/mozdef/envs/mozdef
+    $ source bin/activate
+    $ cd bin
+    $ pip install supervisor
+
+
 Start Services
 ***************
 
-Start the following services ::
+To start the following services you can place the init scripts under /etc/init.d/ and set them to executable. You can find the init scripts in the MozDef/initscripts directory. Or you can start them manually.
 
-    $ invoke-rc.d rabbitmq-server start
+    The initscripts included will match the following startup commands:
 
-    $ service elasticsearch start
+    1. /etc/init.d/rabbitmq-server start or systemctl start rabbitmq-server
 
-    $ service nginx start
+       $ invoke-rc.d rabbitmq-server start
 
-    $ uwsgi --socket /run/uwsgi/apps/loginput.socket --wsgi-file $MOZDEF_PATH/loginput/index.py --buffer-size 32768 --master --listen 100 --uid root --pp $MOZDEF_PATH/loginput --chmod-socket --logto /var/log/mozdef/uwsgi.loginput.log -H $PATH_TO_VENV
+    2. /etc/init.d/elasticsearch start or systemctl start elasticsearch
 
-    $ uwsgi --socket /run/uwsgi/apps/rest.socket --wsgi-file $MOZDEF_PATH/rest/index.py --buffer-size 32768 --master --listen 100 --uid root --pp $MOZDEF_PATH/rest --chmod-socket --logto /var/log/mozdef/uwsgi.rest.log -H $PATH_TO_VENV
+       $ service elasticsearch start
 
-    $ cd $MOZDEF_PATH/mq && uwsgi --socket /run/uwsgi/apps/esworker.socket --mule=esworker.py --mule=esworker.py --buffer-size 32768 --master --listen 100 --uid root --pp $MOZDEF_PATH/mq --stats 127.0.0.1:9192  --logto /var/log/mozdef/uwsgi.esworker.log --master-fifo /run/uwsgi/apps/esworker.fifo -H $PATH_TO_VENV
+    3. /etc/init.d/nginx start or systemctl start nginx
 
-    $ cd $MOZDEF_PATH/meteor && meteor run
+       $ service nginx start
```

Comment:
@Phrozyn 
shouldnt it be:
3.  /etc/init.d/nginx start or systemctl start nginx
$ /etc/init.d/nginx start
$ systemctl start nginx

instead?
(mainly "service" is only available on some distros anyway like RH/CentOS, but also just to be consistent in the doc - theres another instance above for elastisearch)